src/components/ImageBase/ImageBase.tsx

```diff
@@ -14,6 +14,8 @@ export interface ImageBaseProps extends Partial<ImageObjectProps> {
 export const ImageBase = (props: ImageBaseProps) => {
     const {Image} = React.useContext(ImageContext);
 
+    // TODO fix in https://github.com/gravity-ui/page-constructor/issues/964
```

Comment:
`props` has `alt` property already, so it is sufficient to disable the eslint rule